projects/plugins/jetpack/modules/comments/comments.php

```diff
@@ -313,14 +313,15 @@ public function comment_form_after() {
 			 *
 			 * @param string $var Default is "Leave a Reply to %s."
 			 */
-			'greeting_reply'         => apply_filters(
+			'greeting_reply'                               => apply_filters(
 				'jetpack_comment_form_prompt_reply',
 				/* translators: %s is the displayed username of the post (or comment) author */
 				__( 'Leave a Reply to %s', 'jetpack' )
 			),
-			'color_scheme'           => get_option( 'jetpack_comment_form_color_scheme', $this->default_color_scheme ),
-			'lang'                   => get_locale(),
-			'jetpack_version'        => JETPACK__VERSION,
+			'color_scheme'                                 => get_option( 'jetpack_comment_form_color_scheme', $this->default_color_scheme ),
+			'lang'                                         => get_locale(),
+			'jetpack_version'                              => JETPACK__VERSION,
+			'is_test_blockratize_comments_sticker_enabled' => has_blog_sticker( 'test-blockratize-comments', Jetpack_Options::get_option( 'id' ) ),
```

Comment:
It's too verbose IMO. 

```suggestion
			'should_blockratize' => has_blog_sticker( 'test-blockratize-comments', Jetpack_Options::get_option( 'id' ) ),
```